docs/zkapps/roadmap.mdx

```diff
@@ -0,0 +1,97 @@
+---
+title: zkApps Roadmap
+hide_title: true
+sidebar_label: Roadmap
+---
+
+import Subhead from '@site/src/components/common/Subhead';
+
+:::info
+
+Please note that zkApp programmability is not yet available on Mina Mainnet, but
+zkApps can now be deployed to Berkeley Testnet.
+
+:::
+
+# zkApps Roadmap
+
+<Subhead>High-level overview of features available now, next, or later</Subhead>
+
+## Now
```

Comment:
Suggestion: Add a small subtitle or explanation of what "now" and "next" and "later" means.... example, "These are the features that are available now"

Reply:
Do we have a good way to show this typographically? I was hoping the subhead would cover that objective